part-2-server/server.js

```diff
@@ -0,0 +1,50 @@
+var http = require('http');
+var fs= require('fs');
+
+var server = http.createServer(function(request, response){
+
+	
+		if(request.url === '/'){
+			fs.readFile('index.html', function(err,data){
+				if(err){
+				console.log(err)
+			} else {
+				response.write(data.toString())
+				response.end()
+				}
+			})
+		} else if (request.url === '/page-2'){
+				fs.readFile('page-2.html', function(err,data){
+				if(err){
+				console.log(err)
+			} else {
+				response.write(data.toString())
+				response.end()
+				}
+			})
+		} else if(request.url === '/page-3'){
+				fs.readFile('page-3.html', function(err,data){
+				if(err){
+				console.log(err)
+			} else {
+				response.write(data.toString())
+				response.end()
+				}
+			})//creat a loop
+		} else if (request.url === '/poem'+ i){
```

Comment:
`i` isn't declared so it isn't referring to anything here....we'd either need to have a for loop that initializes i, or solve this without using a for loop